pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -96,7 +96,7 @@ var (
 	// https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
 	InternalLBAnnotations = map[configv1.PlatformType]map[string]string{
 		configv1.AWSPlatformType: {
-			awsInternalLBAnnotation: "0.0.0.0/0",
```

Comment:
Would it be worth dropping a comment here explaining that we used to set the annotation to `0.0.0.0/0` in prior OCP versions?

Reply:
Good idea. I've updated the PR with an explanation comment.